portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
```

Comment:
So first of all, not sure why they want you to use BlobStore since the first line in the API (https://cloud.google.com/appengine/docs/standard/java/blobstore)  is, "Note: You should consider using Google Cloud Storage rather than Blobstore for storing blob data." lol... But yeah, whatever, if that's what the instructions say, just use Blobstore.  I have a feeling it's because the setup is simpler.

Reply:
Yeah, idk... If Google Cloud Storage is preferred, then what is the purpose of Blobstore?